guide/start/running.md

```diff
@@ -133,9 +134,9 @@ INFO  Started Brooklyn console at http://127.0.0.1:8081/, running classpath://br
 
 By default it can be accessed by opening [127.0.0.1:8081](http://127.0.0.1:8081){:target="_blank"} in your web browser. 
 
-The rest of this getting started guide uses the Apache Brooklyn command line interface (CLI). To use this, download and install the tool as described on the [CLI GitHub page](https://github.com/brooklyncentral/brooklyn-cli){:target="_blank"}.
+The rest of this getting started guide uses the Apache Brooklyn command line interface (CLI) tool, `br`. 
+This tool is distributed along with Apache Brooklyn. See details on the [Client CLI Reference](../../ops/cli/) page. 
```

Comment:
In follow up to [this](https://github.com/apache/brooklyn-docs/pull/25#discussion_r55657979) comment, I think this URL is incorrect